### PR TITLE
Jenkins: archive Ginkgo test results in each job

### PR DESCRIPTION
**Pull request by ianvernon**
Signed-off by: Ian Vernon <ian@cilium.io>

### CODE REVIEW

**Comment by ianvernon** [comment]
I swapped the Jenkinsfiles so that `ginkgo.Jenkinsfile` would be used as `Jenkinsfile`, since that is what the configuration on Jenkins picks up for running PRs. While the build failed, the archive was created: https://jenkins.cilium.io/job/cilium/job/cilium/job/PR-2326/1/ .

**Review by aanm — APPROVED** [review]

File: test/archive_test_results.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+ 
+cd ${WORKSPACE}/test
+
+tar czf test_results_${JOB_BASE_NAME}_${BUILD_NUMBER}.tar test_results/
```
Comment (eloycoto) [inline test/archive_test_results.sh]: Could you create a variable for `test_results` and avoid the hardcoding in lines 5 and 7. Thanks.

**Review by eloycoto — APPROVED** [review]
Minor comment but can be merged.

File: test/archive_test_results.sh (hunk 1 above)
Comment (ianvernon) [reply test/archive_test_results.sh]: Good idea, will do :)

**Comment by ianvernon** [comment]
I'm surprised this requires janitor review; `CODEOWNERS` shows `test/` as being owned by cilium/ci , and `Jenkinsfile` as owned by cilium/ci as well. Is there a way to see in GitHub what triggered a CODEOWNER to get added to the review?

File: test/archive_test_results.sh (hunk 1 above)
Comment (ianvernon) [reply test/archive_test_results.sh]: Fixed!

**Review by joestringer — APPROVED** [review]

**Pull request merged**